judge/views/register.py

```diff
@@ -33,8 +33,12 @@ class CustomRegistrationForm(RegistrationForm):
     if newsletter_id is not None:
         newsletter = forms.BooleanField(label=_('Subscribe to newsletter?'), initial=True, required=False)
 
-    if ReCaptchaField is not None:
-        captcha = ReCaptchaField(widget=ReCaptchaWidget())
+    if CaptchaField is not None:
+        if ReCaptchaWidget is not None:
+            captcha = CaptchaField(widget=ReCaptchaWidget())
```

Comment:
Do we even need this special construction? https://github.com/DMOJ/django-recaptcha2/blob/master/snowpenguin/django/recaptcha2/fields.py#L22 seems to imply that `widget=ReCaptchaWidget()` is pointless.

Honestly, at this rate, I am tempted to drop ReCAPTCHA soon, given all the problems I ran into with them and also the fact I had to fork the library...